couler/core/templates/artifact.py

```diff
@@ -98,14 +94,23 @@ def to_yaml(self):
             config.update({"bucket": self.bucket})
         if self.endpoint:
             config.update({"endpoint": self.endpoint})
-        yaml_output = OrderedDict(
-            {"name": self.id, "path": self.path, self.type: config}
+        yaml_output = (
+            OrderedDict(
+                {"name": self.id, "path": self.path, self.type: config}
+            )
+            if self.type != "local"
```

Comment:
Move type to a constant so that we can also validate whether the type is supported?

Reply:
@terrytangyuan done, did this for the other types as well.